src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,77 @@
+
+        while(true){
+
+            line = in.nextLine();
+            try {
+                if(!(line.contains("bye")||line.contains("todo")||line.contains("deadline")||line.contains("event")
+                        ||line.contains("list")||line.contains("done"))){
+                    throw new DukeException();
+                }
+                if (line.equals("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!\n");
+                    break;
+                } else if (line.contains("todo")) {
+                    try {
+                        if (line.length() == 4) {
+                            throw new DukeException();
+                        }
+                        String task = line.substring(5, line.length());
+                        tasks[count] = new ToDos(task);
+                        System.out.println("Got it. I've added this task");
+                        System.out.print("  ");
+                        tasks[count++].printStatus();
+                        System.out.println("Now you have " + count + " tasks in the list");
+                    } catch (DukeException e) {
+                        System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    }
+                } else if (line.contains("deadline")) {
+                    String task = line.substring(9, line.indexOf("/"));
+                    String by = line.substring(line.indexOf("/") + 4, line.length());
+                    tasks[count] = new DeadLines(task, by);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.print("  ");
+                    tasks[count++].printStatus();
+                    System.out.println("Now you have " + count + " tasks in the list");
+                } else if (line.contains("event")) {
+                    String task = line.substring(6, line.indexOf("/"));
+                    String by = line.substring(line.indexOf("/") + 4, line.length());
+                    tasks[count] = new Events(task, by);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.print("  ");
+                    tasks[count++].printStatus();
+                    System.out.println("Now you have " + count + " tasks in the list:");
+                } else if (line.contains("list")) {
+                    for (int i = 0; i < count; i++) {
+                        System.out.print(i + 1 + ".");
+                        tasks[i].printStatus();
+                    }
+                } else if (line.contains("done")) {
+                    int index = Integer.parseInt(line.substring(5, line.length()));
+                    tasks[index - 1].setDone(true);
+                    System.out.println("Nice! I've marked this task as done:");
+                    tasks[index - 1].printStatus();
+                }
+            }catch(DukeException e){
```

Comment:
It would be better if all the space between statements and brackets are consistent.